backend/src/main.js

```diff
@@ -23,63 +23,67 @@ const {AuthService} = require('./services/auth')
 const {RaffleService} = require('./services/raffle')
 
 function start(port) {
-  const postgres = PostgresDB(config.database);
-
-  const userRepo = UserRepo(postgres);
-  userRepo.setupRepo();
-
-  const userModel = UserModel(userRepo);
-
-  const authService = AuthService(userModel); 
-  const userController = UserController(userModel, authService);
-  
-  const itemRepo = ItemRepo(postgres);
-  itemRepo.setupRepo();
-  const itemModel = ItemModel(itemRepo);
-  const itemController = ItemController(itemModel, userModel, authService);
-
-  const raffleService = RaffleService(itemModel, userModel);
-
-  const adminController = AdminController(userModel, itemModel, raffleService);
-
-
-  const app = express();
-  app.disable('x-powered-by');
-  app.use(compression());
-  app.use(morgan('dev'));
-  app.use(
-    cors({
-      origin: ['http://localhost:3000'],
-      credentials: true,
-    }),
-  );
-  app.use(cookieParser());
-  app.use(bodyParser.json({
-    verify: (req, res, buf) => {
-      req.rawBody = buf
-    }
-  }))
-
-  const router = express.Router();
-  router.use('/users', userController);
-  router.use('/items', itemController);
-  router.use('/admin', adminController);
-
-  app.use('/api', router);
-
-  app.listen(port, () => {
-    console.log(`Listening on port ${port}`);
-  });
-
-  // Cron job so every night at midnight, all raffles past a deadline are ended
-  var task = cron.schedule('0 0 * * *', () => {
-    raffleService.checkDeadlines()
-   }, {
-     scheduled: true,
-     timezone: "America/Los_Angeles"
-   });
-
-  task.start();
+	const postgres = PostgresDB(config.database);
+
+	const userRepo = UserRepo(postgres);
+	userRepo.setupRepo();
+
+	const userModel = UserModel(userRepo);
```

Comment:
Looks like we went from 2 spaces to a tab for this whole file, if you can find an easy fix in Sublime to get it back, might be cleaner but not a huge deal if not

Reply:
My vim is set up to do tabs not spaces. I think I auto indented this file. I don't know how to make it spaces instead.